### PR TITLE
To do list add remove implementation

### PR DESCRIPTION
**Pull request by olanikegloria**
In this milestone I learnt to 
Use webpack to bundle JavaScript.
 Use proper ES6 syntax.
Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/modules/utility.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        </li>`;
+  });
+
+  tasks.innerHTML = mappedLists.join('');
+  const taskItems = document.querySelectorAll('.task-items');
+
+  taskItems.forEach((item) => {
+    const activities = item.querySelector('.activities');
+    const dotHover = item.querySelector('.dot-hover');
+    const trashBox = dotHover.querySelector('.fa-regular.fa-trash-can');
```
Comment (OyePriscilla) [inline src/modules/utility.js:41]: - [ ] Kindly take note that the `delete icon` is not yet displayed on the UI, I tried to hover on the the `ellipsis icon` (**the three dots**) but nothing was displayed. Kindly address this in order to have a good mastery of what's expected of you according to the project requirement. 👍

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  text-decoration: line-through;
+}
+
+.delete {
+  background-color: beige;
+}
+
+.delete input {
+  background-color: beige;
+}
```
Comment (OyePriscilla) [inline src/style.css]: - [ ] In accordance to the general requirement, please ensure that you didn't repeat the same line of code (CSS DRY principle). Thanks. 🙏

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @olanikegloria 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implemented `Todos'` `Add` , `Delete` and `Edit` functionalities ✔️ 
Saved `Todos` in localstorage ✔️ 

## Required Changes ♻️

- [ ] **Please kindly ensure that the Live Demo header of the README file redirects to the appropraite page, this will make your repo to be engaging and professional. Thanks.** 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,171 @@
...
+
+- **Responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```
Comment (uwadonat) [inline README.md]: - [ ] As requested by the previous reviewer, let's show the recruiter that you are able to pay attention to details:
 - By making sure the live demo is pointing to the deployed application or remove it from there if the project is not deployed.
 - You wrote one feature twice. remove duplication and add another one if any.
error

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @olanikegloria , :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

### To highlight! :+1: 
- Your commit messages are very descriptive ✔️
- Good project structure ✔️
almost

 ## Status: Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @olanikegloria   👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Successfully implement the CRUD (create, update, delete) methods👍
✔ A function for editing task descriptions is implemented. 👍 
✔ A function for deleting a task (remove an element from the array) is implemented👍
✔ All changes to the To-Do List should be saved in local storage.👍

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**